newIDE/app/src/ObjectsRendering/Renderers/CustomObjectLayoutingModel.js

```diff
@@ -85,43 +85,65 @@ const layoutFields = [
   'IsScaledProportionally',
 ];
 
+export const getProportionalPositionX = (positionName: string): ?number => {
+  const horizontalPositionName = (positionName.includes('-')
+    ? positionName.split('-')[1]
+    : positionName
+  ).toLowerCase();
+  return horizontalPositionName === 'left'
+    ? 0
+    : horizontalPositionName === 'right'
+    ? 1
+    : horizontalPositionName === 'center'
+    ? 0.5
+    : null;
+};
+
+export const getProportionalPositionY = (positionName: string): ?number => {
+  const verticalPositionName = (positionName.includes('-')
+    ? positionName.split('-')[0]
+    : positionName
+  ).toLowerCase();
+  return verticalPositionName === 'top'
+    ? 0
+    : verticalPositionName === 'bottom'
+    ? 1
+    : verticalPositionName === 'center'
+    ? 0.5
+    : null;
+};
+
 const getHorizontalAnchorValue = (
   anchorName: string,
-  properties: ?gdMapStringPropertyDescriptor
+  properties: gdMapStringPropertyDescriptor
 ): ?number => {
   const horizontalAnchorName = (anchorName.includes('-')
```

Comment:
doing it here is not needed anymore as you're doing it in the function `getProportionalPositionX`?